lib/fog/storage/google_json/requests/list_objects.rb

```diff
@@ -0,0 +1,50 @@
+module Fog
```

Comment:
Sorry, I'm not following you here, bot. Anyone know what he wants me to `freeze`?